client/state/admin-color/reducer.js

```diff
@@ -1,12 +1,14 @@
 import { withStorageKey } from '@automattic/state-utils';
-import { ADMIN_COLOR_RECEIVE } from 'calypso/state/action-types';
+import { ADMIN_COLOR_RECEIVE, ADMIN_COLOR_REQUEST_FAILURE } from 'calypso/state/action-types';
 import { keyedReducer, withPersistence } from 'calypso/state/utils';
 import 'calypso/state/data-layer/wpcom/sites/admin-color';
 
 const adminColorReducer = ( state = [], action ) => {
 	switch ( action.type ) {
 		case ADMIN_COLOR_RECEIVE:
 			return action.adminColor;
+		case ADMIN_COLOR_REQUEST_FAILURE:
+			return action.adminColor || 'default'; // fallback color scheme: packages/calypso-color-schemes/src/shared/color-schemes/_default.scss
```

Comment:
Is there any issue if we leave it undefined?

Reply:
Yes, there is. It will think that we still haven't fetched the admin color, and we will never show the sidebar 🥲

Reply:
Oh yeah this is the quick way to determine whether the admin color is fetched or not 🫠